src/pages/CreatePool/index.tsx

```diff
@@ -25,13 +26,14 @@ import ReviewCreatePool from "./CreatePoolDialog"
 import { Link as RouteLink } from "react-router-dom"
 import { getContract } from "../../utils"
 import { useActiveWeb3React } from "../../hooks"
+import { usePoolRegistry } from "../../hooks/useContract"
 import { useTranslation } from "react-i18next"
 
-type PoolType = "usdMetapool" | "btcMetapool" | "basepool"
+export type PoolType = "usdMetapool" | "btcMetapool" | "basepool"
 
-type AssetType = "USD" | "ETH" | "BTC" | "OTHERS"
+export type AssetType = 0 | 1 | 2 | 3
```

Comment:
please use enum here. Will give you the best of both worlds `AssetType.ETH == 1`